<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPatDialog</class>
 <widget class="QDialog" name="AddPatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>472</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>339</width>
    <height>472</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>339</width>
    <height>472</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AddPatDialog{
background-color: qradialgradient(spread:pad, cx:0.494318, cy:0.5, radius:0.859, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(222, 227, 241, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">color: rgb(13, 111, 122);
font: 75 26pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:26pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Quantify'; font-size:28pt; font-weight:400;&quot;&gt;Nouveau patient&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="numPat">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Quantify&quot;;
color:rgb(232, 54, 54);</string>
     </property>
     <property name="text">
      <string>PAT-4</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nomInput">
         <property name="minimumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid qlineargradient(spread:pad, x1:0, y1:0.534, x2:0.994318, y2:0.518, stop:0 rgba(144, 250, 197, 255), stop:1 rgba(132, 165, 253, 255));
border: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 34, 34, 255), stop:1 rgba(55, 198, 214, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0.989, x2:1, y2:0, stop:0 rgba(222, 237, 255, 198), stop:1 rgba(255, 255, 255, 255));
font-size: 11pts;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="prenomInput">
         <property name="minimumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid qlineargradient(spread:pad, x1:0, y1:0.534, x2:0.994318, y2:0.518, stop:0 rgba(144, 250, 197, 255), stop:1 rgba(132, 165, 253, 255));
border: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 34, 34, 255), stop:1 rgba(55, 198, 214, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0.989, x2:1, y2:0, stop:0 rgba(222, 237, 255, 198), stop:1 rgba(255, 255, 255, 255));
font-size: 11pts;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Sexe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="sexeCombo">
         <property name="minimumSize">
          <size>
           <width>211</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>211</width>
           <height>81</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Masculin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Feminin</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Adresse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="adresseInput">
         <property name="minimumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid qlineargradient(spread:pad, x1:0, y1:0.534, x2:0.994318, y2:0.518, stop:0 rgba(144, 250, 197, 255), stop:1 rgba(132, 165, 253, 255));
border: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 34, 34, 255), stop:1 rgba(55, 198, 214, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0.989, x2:1, y2:0, stop:0 rgba(222, 237, 255, 198), stop:1 rgba(255, 255, 255, 255));
font-size: 11pts;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addMedBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#addMedBtn{
	border:none;
border: 1px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 59, 59, 255), stop:1 rgba(98, 202, 214, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 59, 59, 255), stop:1 rgba(98, 202, 214, 255));
color: rgb(232, 54, 54);
color: white;
font-weight: bold;
}
#addMedBtn:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 34, 34, 255), stop:1 rgba(55, 198, 214, 255));
}</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#cancelBtn{
border:none;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(11, 102, 112, 255), stop:1 rgba(18, 185, 200, 255));
color: rgb(232, 54, 54);
color: white;
font-weight: bold;
}
#cancelBtn:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(9, 87, 94, 255), stop:1 rgba(15, 174, 182, 255));
}</string>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
